pom.xml

```diff
@@ -115,6 +116,12 @@
       <version>${metrics-healthchecks-severity.version}</version>
     </dependency>
 
+    <dependency>
```

Comment:
This one should, dropwizard-client was really only needed for the tests because we created the Client.  Anyone using this will already have whatever is needed for the client.